src/modules/tracker.js

```diff
@@ -296,16 +296,21 @@ class Tracker {
         const url = `${this.options.serviceUrl}/behavior?`;
         const queryParams = { action: 'search-results', term };
         const { num_results, customer_ids, item_ids } = parameters;
+        let truncatedCustomerIDs;
 
         if (!helpers.isNil(num_results)) {
           queryParams.num_results = num_results;
         }
 
         // Ensure support for both item_ids and customer_ids as parameters
         if (item_ids && Array.isArray(item_ids) && item_ids.length) {
-          queryParams.customer_ids = item_ids.join(',');
+          truncatedCustomerIDs = item_ids;
         } else if (customer_ids && Array.isArray(customer_ids) && customer_ids.length) {
-          queryParams.customer_ids = customer_ids.join(',');
+          truncatedCustomerIDs = customer_ids;
+        }
+
+        if (truncatedCustomerIDs && Array.isArray(truncatedCustomerIDs) && truncatedCustomerIDs.length) {
```

Comment:
Probably don't need to check that it's an array again since we're already doing it above. Might just be able to simplify to `truncatedCustomerIDs?.length`. Will let you decide. 🙂